src/compile.ts

```diff
@@ -470,7 +552,7 @@ function getOperator(op: ts.BinaryOperatorToken): BinaryOp | undefined {
   return OperatorMappings[op.kind as keyof typeof OperatorMappings];
 }
 
-const OperatorMappings = {
+const OperatorMappings: Record<number, BinaryOp> = {
```

Comment:
Why explicit type? I think I was purposely using an inferred type.

Reply:
So the compiler fails when the operator isn't supported by `BinaryOp` so that other places have a chance to handle the value. For example, I added `!=` in #39, but function that is trying to handle all BinaryOps may be missed if it doesn't know about the new operator. 

Note: this was pulled forward from #39, could remove it from this PR.